app/controllers/users_controller.rb

```diff
@@ -1,7 +1,22 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
+  before_action :load_user, only: [:show, :edit, :update, :destroy]
+
+  def destroy
+    if @user.destroy
+      flash[:success] = t "users.text_5"
+    else
+      flash[:danger] = t "users.text_6"
+    end
+    redirect_to users_url
+  end
+
+  def index
+   @pagy, @users = pagy(User.all)
```

Comment:
muốn setting mỗi trang có 10 bản ghi thì làm tnao e

Reply:
dạ nếu muốn áp dụng cho toàn bộ những nơi có sử dụng pagy thì tạo file pagy.rb rồi thêm  : Pagy::VARS[:items] = 25
Nếu muốn thay đỗi 1 chổ nào đó thì dùng thêm 1 option là  items:value
 @pagy, @users = pagy(User.all, items: 30)